Update Security policy to be more precise about vulns (#1411)
rizinorg/rizin: SECURITY.md

# Security Policy

## Scope and Supported Versions

| Version          | Supported          |
| ---------------- | ------------------ |
| latest-release   | :white_check_mark: |
| *                | :x:                |

> Rizin is a user-space Command Line Interface (CLI) tool which isn't considered a high-availability service. As such, issues that impact Rizin's availability will not be considered as vulnerbilities by the security team. Such issues include, but not limitied to:

Reviewer: vulnerabilities

Reviewer 2: ```suggestion
Rizin is a user-space Command Line Interface (CLI) tool that isn't considered a high-availability service. As such, issues that impact Rizin's availability will not be considered as vulnerabilities by the security team. Such issues include, but are not limited to:
```

 - NULL Pointer Dereference
 - Memory Exhaustion

> Issues that can leak non-sensitive memory contents (e.g memory addresses and values, etc.) shuld be demonstrated to be usable by an attacker, and will be considered on a case-by-case basis.

Reviewer: should

Reviewer: Please no `case-by-case basis`, it's a poor style.

Reviewer 2: ```suggestion
Issues that can leak non-sensitive memory contents (e.g memory addresses and values, etc.) should be demonstrated to be usable by an attacker and will be considered on a case-by-case basis.
```

- Inform the team over the [public chats](https://rizin.re/#community) that you sent a message regarding a security issue.

**Important:** Don't disclose any information regarding the issue itself in the public chats.

> Given the current usage of Rizin and its libraries as a standalone Command Line Interface (CLI) tool, the security team does not consider as vulnerabilities those issues that simply crash rizin or one of the other tools (e.g. NULL pointer dereferences). Issues that can leak non-sensitive memory contents (e.g. memory addresses/memory values that are not files, private keys, etc.) will be considered on a case-by-case basis and they should be demonstrated to be somehow usable by an attacker.

Reviewer: I would add it's a `user-space tool` and **not a service**.

> While analysing reports, we consider the main usage of Rizin nowadays as a user-space Command Line Interface (CLI) tool and not as a service or as a library. For this reason the security team does not consider as vulnerabilities those issues that simply crash rizin or one of the other tools (e.g. NULL pointer dereferences, memory exhaustion or in general those flaws that only impact a service availability). Issues that can leak non-sensitive memory contents (e.g. memory addresses/memory values that are not files, private keys, etc.) will be considered on a case by case and they should be demonstrated to be somehow usable by an attacker. Please try to always include in your report an attack scenario, showing how the issue can affect Rizin users. 

Reviewer: `considered on a case by case` -> `will be considered case by case`

> While analysing reports, we consider the main usage of Rizin nowadays as a user-space Command Line Interface (CLI) tool and not as a service or as a library. For this reason the security team does not consider as vulnerabilities those issues that simply crash rizin or one of the other tools (e.g. NULL pointer dereferences, memory exhaustion or in general those flaws that only impact a service availability). Issues that can leak non-sensitive memory contents (e.g. memory addresses/memory values that are not files, private keys, etc.) will be considered on a case by case and they should be demonstrated to be somehow usable by an attacker. Please try to always include in your report an attack scenario, showing how the issue can affect Rizin users. 

Reviewer 3: ```suggestion
While analysing reports, we consider the main usage of Rizin nowadays as a user-space Command Line Interface (CLI) tool and not as a service or as a library. For this reason the security team does not consider as vulnerabilities those issues that simply crash rizin or one of the other tools (e.g. NULL pointer dereferences, memory exhaustion or in general those flaws that only impact a service availability). Issues that can leak non-sensitive memory contents (e.g. memory addresses/memory values that are not files, private keys, etc.) will be considered case by case and they should be demonstrated to be somehow usable by an attacker. Please try to always include in your report an attack scenario, showing how the issue can affect Rizin users. 
```
